src/util/domain-icon.js

```diff
@@ -38,6 +38,9 @@ export default function domainIcon(domain, state) {
     case 'lock':
       return state && state === 'unlocked' ? 'mdi:lock-open' : 'mdi:lock';
 
+    case 'garage_door':
+      return state && state === 'open' ? 'mdi:arrow-up' : 'mdi:arrow-down';
```

Comment:
Well, anything that resembles something of a garage door would be better.
Arrows are meant for actions. What about mdi:glassdoor ?

On Sun, Jan 31, 2016 at 4:30 PM, w1ll1am23 <notifications@github.com> wrote:

> In src/util/domain-icon.js
> <https://github.com/balloob/home-assistant-polymer/pull/26#discussion_r51370947>
> :
>
> > @@ -38,6 +38,9 @@ export default function domainIcon(domain, state) {
> >      case 'lock':
> >        return state && state === 'unlocked' ? 'mdi:lock-open' : 'mdi:lock';
> >
> > +    case 'garage_door':
> > +      return state && state === 'open' ? 'mdi:arrow-up' : 'mdi:arrow-down';
>
> Agreed, but not sure what would be any better?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant-polymer/pull/26/files#r51370947>
> .
>



-- 
PaulusSchoutsen.nl
It's nice to be important but it's more important to be nice.